File: PlaceYourBets/FixtureBox.vb
﻿Public Class FixtureBox

    Public Sub New(home As String, away As String, fixtureDate As Date, homeBadge As Image, awayBadge As Image)

        ' This call is required by the designer.
        InitializeComponent()

        ' Add any initialization after the InitializeComponent() call.
        homeLabel.Text = home
        awayLabel.Text = away
        dateLabel.Text = fixtureDate
        homeBadgeBox.Image = homeBadge
        awayBadgeBox.Image = awayBadge

    End Sub


    Private Sub nextButton_Click(sender As System.Object, e As System.EventArgs) Handles nextButton.Click
        If ContainerForm.getFixtureNo() = 9 Then
            ContainerForm.reviewPredictions()
        Else
            ContainerForm.makeFixtureVisible(True)
        End If
    End Sub

    Private Sub backButton_Click(sender As System.Object, e As System.EventArgs) Handles backButton.Click
        ContainerForm.makeFixtureVisible(False)
    End Sub

End Class

File: PlaceYourBets.ConvertedToC#/FixtureBox.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
namespace PlaceYourBets
{
	public partial class FixtureBox
	{


		public FixtureBox(string home, string away, System.DateTime fixtureDate, Image homeBadge, Image awayBadge)
		{
			// This call is required by the designer.
			InitializeComponent();

			// Add any initialization after the InitializeComponent() call.
			homeLabel.Text = home;
			awayLabel.Text = away;
			dateLabel.Text = fixtureDate;
			homeBadgeBox.Image = homeBadge;
			awayBadgeBox.Image = awayBadge;

		}


		private void nextButton_Click(System.Object sender, System.EventArgs e)
		{
			if (My.MyProject.Forms.ContainerForm.getFixtureNo() == 9) {
				My.MyProject.Forms.ContainerForm.reviewPredictions();
			} else {
				My.MyProject.Forms.ContainerForm.makeFixtureVisible(true);
			}
		}

		private void backButton_Click(System.Object sender, System.EventArgs e)
		{
			My.MyProject.Forms.ContainerForm.makeFixtureVisible(false);
		}

	}
}
